client/organization.go

```diff
@@ -4,26 +4,40 @@ import (
 	"errors"
 )
 
-func (self *ApiClient) Organization() (Organization, error) {
+func (ac *ApiClient) Organization() (Organization, error) {
```

Comment:
gostaticcheck complains about "self".
I was working in this file, so changed it.